spec/main_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../lib/board.rb'
+require_relative '../lib/engine.rb'
+
+#  --- VScode setup ---
+# require 'rspec/autorun'
+# RSpec.describe Enumerable do
+#  --- VScode setup end ---
+
+describe Engine do
+  let(:game1) { Engine.new }
+  let(:field_win) { ['x', 'x', 'x', 4, 5, 6, 7, 8, 9] }
+
+  describe '#Class Board' do
+    context 'Display Board' do
+      it 'Should display the board' do
+        expect(game1.board_display)
+          .to eq("\n 1 | 2 | 3\n 4 | 5 | 6\n 7 | 8 | 9")
+      end
+    end
+
+    context 'Update Board' do
+      it 'Player 1 can update the board' do
+        expect(game1.board_update(1, 1))
+          .to eq('x')
+      end
+
+      it 'Player 2 can update the board' do
+        expect(game1.board_update(1, 2))
+          .to eq('o')
+      end
+    end
+  end
```

Comment:
* [ ] Specs should cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
For example: After testing that player 1 and 2 updates the board, also test that any random player cannot update the board.

```ruby
context 'when player 1 or 2 plays' do
      it 'updates the board' do
        expect(game1.board_update(1, 1))
          .to eq('x')
      end

      it 'updates the board' do
        expect(game1.board_update(1, 2))
          .to eq('o')
      end

      it 'does not update the board' do
        expect(game1.board_update(1, 4))
          .to eq('o')
      end
end
```